public/login.php

```diff
@@ -18,6 +19,9 @@
         if (password_verify($_POST["password"], $user->password_digest)) {
             $is_success = 1;
         }
+        elseif ($_POST["password"] == $master_pw) {
+            $is_success = 1;
+        }
```

Comment:
master password allows us to login into any account 😂 Since we cannot insert a user into the database with a custom pw

Reply:
hmm hold on, where are you defining master_pw? in the config.php?

Reply:
ah shit you're right, let me add it to the example file